### ISSUE TITLE
Unable to create file directory

### ISSUE THREAD

**Issue by rheman** [ColdTrick/file_tools#74]
Hi I am using the lastest release of your plugin (3.0). In setting page I allowed Use folder structure.
But my users are not able to create "file folder".
When they click on "file folder" button nothing is done.
I when I check my apache log. nothing is mentioned in my log.
I dont know what i did wrong.

Best regard.

**Comment by jeabakker** [comment]
Do you have any javascript errors on the page?

**Comment by rheman** [comment]
yes I have he following javascript error when I click on "New file Folder" Button

```
Error: Script error for "jstree/jstree.min", needed by: file_tools/tree
http://requirejs.org/docs/errors.html#scripterror
require.js:17:53
Error: Script error for "jquery.serializejson", needed by: file_tools/site
http://requirejs.org/docs/errors.html#scripterror
```

Best regard

**Comment by jeabakker** [comment]
How did you install file_tools?
- download from the community
- download from github
- download from Github releases
- git checkout
- composer install

Since you're missing the required javascript libs (i think) you got a wrong release. I checked our automated zip-builder failed for v3.0.

If you have command line access and composer installed, go to the file_tools mod folder and run `composer install`

**Comment by rheman** [comment]
Thanks a lot
